### ISSUE TITLE
No output

### ISSUE THREAD

**Issue by willianveiga** [prontolabs/pronto-brakeman#17]
I've created a new Rails project, I made an initial commit, and I've installed `pronto` and `pronto-brakeman`:

```
# ...
group :development do
  # ...
  gem 'pronto'
  gem 'pronto-brakeman', require: false
end
# ...
```

And then: `$ bundle`.

I've deleted the `end` keyword in `app/controllers/application_controller.rb` without commiting.

When I execute `brakeman` I can see the following output:

```
Loading scanner...
Processing application in testing-app
Processing gems...
[Notice] Detected Rails 5 application
Processing configuration...
[Notice] Escaping HTML by default
Parsing files...
Processing initializers...
Processing libs...ed
Processing routes...          
Processing templates...       
Processing data flow in templates...
Processing models...          
Processing controllers...     
Processing data flow in controllers...
Indexing call sites...        
Running checks in parallel...
 - CheckBasicAuth
 - CheckBasicAuthTimingAttack
 - CheckCrossSiteScripting
 - CheckContentTag
 - CheckCreateWith
 - CheckDefaultRoutes
 - CheckDeserialize
 - CheckDetailedExceptions
 - CheckDigestDoS
 - CheckDynamicFinders
 - CheckEscapeFunction
 - CheckEvaluation
 - CheckExecute
 - CheckFileAccess
 - CheckFileDisclosure
 - CheckFilterSkipping
 - CheckForgerySetting
 - CheckHeaderDoS
 - CheckI18nXSS
 - CheckJRubyXML
 - CheckJSONEncoding
 - CheckJSONParsing
 - CheckLinkTo
 - CheckLinkToHref
 - CheckMailTo
 - CheckMassAssignment
 - CheckMimeTypeDoS
 - CheckModelAttrAccessible
 - CheckModelAttributes
 - CheckModelSerialize
 - CheckNestedAttributes
 - CheckNestedAttributesBypass
 - CheckNumberToCurrency
 - CheckPermitAttributes
 - CheckQuoteTableName
 - CheckRedirect
 - CheckRegexDoS
 - CheckRender
 - CheckRenderDoS
 - CheckRenderInline
 - CheckResponseSplitting
 - CheckRouteDoS
 - CheckSafeBufferManipulation
 - CheckSanitizeMethods
 - CheckSelectTag
 - CheckSelectVulnerability
 - CheckSend
 - CheckSendFile
 - CheckSessionManipulation
 - CheckSessionSettings
 - CheckSimpleFormat
 - CheckSingleQuotes
 - CheckSkipBeforeFilter
 - CheckSQL
 - CheckSQLCVEs
 - CheckSSLVerify
 - CheckStripTags
 - CheckSymbolDoSCVE
 - CheckTranslateBug
 - CheckUnsafeReflection
 - CheckValidationRegex
 - CheckWithoutProtection
 - CheckXMLDoS
 - CheckYAMLParsing
Checks finished, collecting results...
Generating report...

== Brakeman Report ==

Application Path: testing-app
Rails Version: 5.2.2
Brakeman Version: 4.3.1
Scan Date: 2018-12-13 07:15:29 -0200
Duration: 0.051262497 seconds
Checks Run: BasicAuth, BasicAuthTimingAttack, ContentTag, CreateWith, CrossSiteScripting, DefaultRoutes, Deserialize, DetailedExceptions, DigestDoS, DynamicFinders, EscapeFunction, Evaluation, Execute, FileAccess, FileDisclosure, FilterSkipping, ForgerySetting, HeaderDoS, I18nXSS, JRubyXML, JSONEncoding, JSONParsing, LinkTo, LinkToHref, MailTo, MassAssignment, MimeTypeDoS, ModelAttrAccessible, ModelAttributes, ModelSerialize, NestedAttributes, NestedAttributesBypass, NumberToCurrency, PermitAttributes, QuoteTableName, Redirect, RegexDoS, Render, RenderDoS, RenderInline, ResponseSplitting, RouteDoS, SQL, SQLCVEs, SSLVerify, SafeBufferManipulation, SanitizeMethods, SelectTag, SelectVulnerability, Send, SendFile, SessionManipulation, SessionSettings, SimpleFormat, SingleQuotes, SkipBeforeFilter, StripTags, SymbolDoSCVE, TranslateBug, UnsafeReflection, ValidationRegex, WithoutProtection, XMLDoS, YAMLParsing

== Overview ==

Controllers: 0
Models: 1
Templates: 2
Errors: 1
Security Warnings: 0

== Warning Types ==


== Errors ==

Error: testing-app/app/controllers/application_controller.rb:2 :: parse error on value "$end" ($end)
Location: Could not parse testing-app/app/controllers/application_controller.rb

No warnings found
```

Executing `pronto run`, `pronto run --unstaged` and `pronto run --staged` does not give any output at all.

**Comment by pconpie** [comment]
I have been encountering the same problem, and after some digging found the culprit to be the `patch_for_warning` method on the Pronto::Brakeman runner.

```
def patch_for_warning(ruby_patches, warning)
  ruby_patches.find do |
    patch.new_file_full_path.to_s == warning.file
  end
end
```
Turns out `warning.file` has an absolute and relative path on it, so I added `warning.file.absolute` and that fixed the problem for me.

**Comment by os6sense** [comment]
Noticed this happening also ... adding a comment in the hope the maintainer notices and we can get a new release.

**Comment by jameskbride** [comment]
Bump. Still seeing this behavior. @mmozuras, any chance of a new release with this fix?

**Comment by mehagar** [comment]
In the meantime, you can work around this by using the `specific_install` gem to install the forked repo that has the fix from #17.

**Comment by Mathiou04** [comment]
@mehagar I still got the buggy version on `v0.10.0` on a fresh install, I had to workaround in my Gemfile by specifying the exact commit like this:
```
gem 'pronto-brakeman', require: false, github: 'prontolabs/pronto-brakeman', ref: '98471a207fb0d07fa235ebf00d6721ee7f783563'
```
Maybe the `v0.10.0` tag needs to be moved so users can pull the right version? Or the gem needs to be republished?

**Comment by ashkulz** [comment]
A new version has been released to RubyGems (0.11.0).
